GSA-TTS/FAC
# 3303 missing information in fac summary report

## Description
See ticket #3303  for details.
The purpose of this ticket was to address the issue of missing field values in the `note` worksheet within the summary report. However, it turned out that the fields were not missing. Instead, this PR moves the single fields from the `note` worksheet into a new `note_coversheet` worksheet and reorders all worksheets alphabetically, ensuring that `note` and `note_coversheet` are adjacent to each other.

## How to Test
1. Launch the application from the main branch and attempt to download any summary report. Observe that all the single and array fields of the NoteToSefa workbook are rendered in the `note` worksheet.
2. Switch to this PR branch and download the same summary report or any other. Notice that a new sheet named `note_coversheet` has been added, which contains only the single fields of NoteToSefa, while the array fields remain in `note`. Also, observe that the sheets are now ordered alphabetically.

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
